### ISSUE TITLE
The Paleobiology Database

### ISSUE THREAD

**Issue by jwikle** [imls-dmt/resources-workflow#24]
Information about registered data management training resource

<<<<<<< HEAD
- [ ] Name of resource: The Paleobiology Database
- [ ] Date resource added: 08/27/2018
- [ ] Registered by: jwikle
- [ ] Clearinghouse URL: http://dmtclearinghouse.esipfed.org/node/10157

Tracking: @karlbenedict
=======
- [ ] Date resource added: 08/27/2018
- [ ] Registered by:  jwikle
- [ ] Clearinghouse URL: http://dmtclearinghouse.esipfed.org/node/10157

**Comment by njhoebel** [comment]
Great find!  I made the following changes to the record:
1.  Changed the title and URL to link to the educational resources associated with the database since the DB was not specifically focused on data management, but rather on acquiring skills as part of the research data lifecycle.  
2.  Added information in the description that explained the educational resources offered via the Resources page (including the apps, tutorials,, curriculum guides, etc.  
3.  Added a couple of keywords to the list and to this resource including:  paleobiology data, fossil data, and also the (existing) R environment for statistical computing.  
4.  Added SW engineer and educator to the target audiences.
5.  Added a timeframe to the resources
6.  Debated changing the learning resource type, but agree that "Service" is the most appropriate for this rich resource, especially as there is a DB associated with it.  
7.  Moved to Publish.
